doc_src/index.rst

```diff
@@ -47,6 +47,10 @@ News
     negative signatures), choice of braid generators (now positive
     generators give positive crossings).
 
+  - Supporting argument `first_cusps` when picking disjoint cusp neighborhoods greedily with :meth:`cusp_areas <snappy.Manifold.cusp_areas>` and :meth:`short_slopes <snappy.Manifold.short_slopes>`. Contributed by Robert Haraway.
```

Comment:
Not sure whether this is a large enough change that it should really be in the news section.

Reply:
I lean toward it being a bit too minor.  Perhaps we should add Robert to `credits.rst` instead?

Reply:
Agreed. I don't even know whether this contribution is large enough for credits.rst.

Reply:
I'm not sure the change to `cusp_translations` should be its own item in the news.  Or rather, there are other changes of similar magnitude not mentioned, like `Link` now accepts `DT: blah` as a spec. 

Maybe we should have a single line such as "updates to methods cusp_translations, cusp_area, short_slopes (and whatever else has changed); Link now accepts "DT: blah" as input".

Reply:
I called out cusp_translations because it is the one breaking change. Maybe it is not important enough. I like your single line suggestion. Maybe replace blah by _.

Reply:
Looking through the commits to see what else has changed:  

- You made some changes to CS in 9bce41b3e111, does that need a mention?
- DirchletDomain.length_spectrum_dicts got a new flag in bbbde47baf2.
- There are changes to "secret" methods and things like Mcomplex, but those are definitely below the radar.

Reply:
> Looking through the commits to see what else has changed:
> 
> * You made some changes to CS in [9bce41b](https://github.com/3-manifolds/SnapPy/commit/9bce41b3e111bcd2291f634eb1275c651c1910a0), does that need a mention?

I don't think so.

> * DirchletDomain.length_spectrum_dicts got a new flag in [bbbde47](https://github.com/3-manifolds/SnapPy/commit/bbbde47baf2775825a1dd88a0c53d583fe3c8103).

I don't think that is the right change.

> * There are changes to "secret" methods and things like Mcomplex, but those are definitely below the radar.

Agreed.

Reply:
Sorry, meant the `grouped` flag in d6e61c27fae1e31.

Reply:
Feels too small for news.

Reply:
Ok, in the interests of time, I will just put something along the lines we're discussing straight into master.